src/components/Clock.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class Clock extends React.Component {
+  state = {
+    date: new Date(),
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState({ date: new Date() });
+    }, 1000);
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
вот можно создать переменную, что приятно сократит код, тем более ты используешь ее 2 раза
```suggestion
 const time = this.state.date.toLocaleTimeString()
```